rebasehelper/tags.py

```diff
@@ -80,7 +80,52 @@ def _map_sections_to_parsed(cls, raw_content: SpecContent, parsed_content: SpecC
 
     @classmethod
     def _parse(cls, raw_content: SpecContent, parsed_content: SpecContent) -> Tuple[Tag]:
-        """Parses all tags from provided SPEC content and determines if they are valid.
+        result = []
+        parsed_mapping = cls._map_sections_to_parsed(raw_content, parsed_content)
+        for section_index, (section, section_content) in enumerate(raw_content.sections):
+            section = section.lower()
+            parsed_index = parsed_mapping[section_index]
+            parsed: List[str] = [] if parsed_index == -1 else parsed_content.sections[parsed_index][1]
+            if section.startswith('%package'):
+                result.extend(cls._parse_package_tags(section, section_content, parsed))
+            elif section.startswith('%sourcelist') or section.startswith('%patchlist'):
```

Comment:
Why `startswith()` here?

Reply:
Currently `%sourcelist` and `%patchlist` don't support any arguments but I think they potentially could in the future so I used `startswith()` just to be safe if that makes sense

Reply:
What arguments would that be? But fine, it doesn't hurt anything.